stylesheets/components/avatar.css

```diff
@@ -0,0 +1,74 @@
+@import url("/stylesheets/main.css");
+
+.avatar {
+  margin: 1rem;
+  width: 3rem;
+  height: 3rem;
+  border-radius: 100%;
+  position: relative;
+}
+
+.avatar-sm {
+  width: 2rem;
+  height: 2rem;
+}
+
+.avatar-lg {
+  width: 3.5rem;
+  height: 3.5rem;
+}
+
+.avatar-xl {
+  width:4rem;
+  height:4rem;
+}
+
+.avatar-sq {
+  border-radius: 0.2rem;
+}
+
+.avatar-container {
+  position: relative;
+  width: fit-content;
+}
+
+.badge-online {
+  width: 15px;
+  height: 15px;
+  border-radius: 100%;
+  background-color: var(--badge-online);
+  position: absolute;
+  bottom: 25px;
+  left: 50px;
```

Comment:
Use percentage instead of a fixed value in pixels, because suppose you created this badge for an avatar of height and width 50px, then it would work fine for that particular size only, but if the same badge would be used for another avatar of different dimensions, it won't be positioned well.